src/main/java/com/epam/spring/homework2/config/BeansConfig.java

```diff
@@ -0,0 +1,14 @@
+package com.epam.spring.homework2.config;
+
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+import org.springframework.context.annotation.PropertySource;
+
+@Configuration
+@ComponentScan("com.epam.spring.homework2.another")
+@ComponentScan("com.epam.spring.homework2.beans")
```

Comment:
small thing, you could combine these packages into single annotation, for ex:
`@ComponentScan({ "pack1", "pack2" })`